Source/CarthageKit/GitHub.swift

```diff
@@ -83,6 +83,15 @@ public struct GitHubRelease: Equatable {
 	/// The name of this release.
 	public let name: String
 
+	/// The name of this release, with fallback to its tag when the name is an emtpy string
```

Comment:
Typo: `emtpy` :arrow_right: `empty`.

Also, could you please end the sentence with a `.`?